config/locales/client.en.yml

```diff
@@ -1667,13 +1667,15 @@ en:
         server: "Server Error"
         forbidden: "Access Denied"
         unknown: "Error"
+        no_data: "Error Not Found"
         not_found: "Page Not Found"
       desc:
         network: "Please check your connection."
         network_fixed: "Looks like it's back."
         server: "Error code: %{status}"
         forbidden: "You're not allowed to view that."
         not_found: "Oops, the application tried to load a URL that doesn't exist."
+        no_data: "Something went wrong, or you may have visited this page directly."
```

Comment:
I don't quite understand what "you many have visited this page directly" means, and I worry if I don't the end user will not! Is there something more actionable we can say?

Reply:
Yeah, I wasn't too sure. This is what gets shown when you visit `/exception`, which was only generated in some intermediate code revisions (I decided against intentionally generating it for now).

Reply:
I'd prefer "Something went wrong" without the other part in that case.

Reply:
Done